hotelReservation/Dockerfile

```diff
@@ -14,17 +14,7 @@ COPY tls/ tls/
 COPY tracing/ tracing/
 COPY tune/ tune/
 
-RUN CGO_ENABLED=0 GOOS=linux GO111MODULE=on go install -ldflags="-s -w" -mod=vendor ./cmd/...
-
-FROM gcr.io/distroless/static:nonroot
```

Comment:
I'm more so curious, is there a reason to remove the distroless image as the final one?